android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -315,6 +315,10 @@ public void testMethodsExist() throws Exception {
                 "insertSiteSectionView", false, null));
         Assert.assertTrue(methodExists("org/chromium/chrome/browser/ntp/NewTabPageLayout",
                 "isScrollableMvtEnabled", true, boolean.class));
+        Assert.assertTrue(methodExists("org/chromium/chrome/browser/ntp/NewTabPageLayout",
+                "isScrollableMvtEnabled", true, boolean.class));
+        Assert.assertTrue(methodExists("org/chromium/chrome/browser/ntp/NewTabPageLayout",
+                "hasLoadCompleted", false, null));
```

Comment:
It likely needs to be whitelisted in `apk_for_test.flags` to work in release. And same applies to `mLogoCoordinator`

Reply:
Added now.